dataframe_api_compat/pandas_standard/__init__.py

```diff
@@ -104,12 +104,15 @@ def map_pandas_dtype_to_standard_dtype(dtype: Any) -> DType:
         return Namespace.Float32()
     if dtype == "Float32":
         return Namespace.Float32()
-    if dtype == "bool":
+    if dtype in ("bool", "boolean"):
```

Comment:
I discovered it by accident while experimenting. It is possible that this is no longer necessary for the current changes.